taxonomies/food/ingredients.txt

```diff
@@ -54261,6 +54312,7 @@ carbon_footprint_fr_foodges_value:fr: 0.3
 ciqual_food_code:en: 20009
 ciqual_food_name:en: Carrot, raw
 ciqual_food_name:fr: Carotte, crue
+ecobalyse:en: carrot-organic
```

Comment:
There's no "carrot" entry in Ecobalyse? If there is, carrot-organic should not be the default value.

Reply:
My bad i had not seen that !